CIP-XXXX/README.md

```diff
@@ -0,0 +1,148 @@
+---
+CIP: ????
```

Comment:
```suggestion
CIP: 129
```

Reply:
and please remember to rename your CIP-XXXX directory

Reply:
done